src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed && 'TodoInfo--completed'}`}>
```

Comment:
The code might break if the 'completed' property is undefined. Consider adding a check to ensure that the 'completed' property is defined.